### PR TITLE
Novice GitHub image fix and change of fig 42

### PR DESCRIPTION
**Pull request by EKaroune**
### Summary

fix squashed images in novice github and chnage fig 42.

Fixes #<NUM>

### List of changes proposed in this PR (pull-request)

- [x] change fig 42 to correct image
- [x] add alt text 
- [x] fix all squashed figures in chapter 

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by EKaroune** [comment]
I have changed the image file and the text in the github features subchapter. I think I did it the same way as @malvikasharan but the build seems to have failed?? not sure why??

**Review by paulowoicho — COMMENTED** [review]
Everything looks great! Just that it's failing the CI test introduced in #1623. Maybe @sgibson91 could help us out here?

**Review by malvikasharan — APPROVED** [review]
I need to update the requirement file with the sphinx version to avoid CI from failing. See this: https://github.com/executablebooks/jupyter-book/issues/1137.

Once it's sorted then you can merge it.

File: book/website/collaboration/github-novice/github-novice-features.md [hunk 1]
```diff
@@ -36,14 +36,22 @@ This is what a repository looks like when it is set up as a collaborative projec
 
 ```{figure} ../../figures/github-project.jpg
 ---
-height: 500px
 name: github-project
-alt: Annotated diagram of README.md file on landing page with template added. Explained in figure legend.
+alt: Annotated diagram of repository after making it into a collaborative project.
```
Comment (malvikasharan) [inline book/website/collaboration/github-novice/github-novice-features.md:40]:
```suggestion
alt: Annotated diagram of the repository after making it into a collaborative project. Explained in the title.
```

**Review by malvikasharan — APPROVED** [review]
Minor update.

**Comment by malvikasharan** [comment]
Thank you @EKaroune and @paulowoicho. This looks pretty good :) Approved for merging.

**Pull request merged**